### DISCUSSION TITLE
[Proposal][iOS] Allow creation of MediaSource from stream / byte[]

### DISCUSSION THREAD

**Discussion by Suprndm** [CommunityToolkit/Maui#1093] [New Feature Discussions]
I'm in the case where i get the media data from API by stream. 
Currently MediaSource allows creation from FilePath or from URI.

Is it possible to also expose a method that takes a Stream ?

I could implement it myself if you tell me which methods to override from MediaSource to allow such behavior.

Thanks

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
